cellprofiler/modules/expandorshrinkobjects.py

```diff
@@ -14,30 +24,14 @@
 otherwise shrinking them, and can perform some specialized morphological
 operations that remove pixels without completely removing an object.
 
-*Note on saving images:* You can pass the modified objects along to the
-*Object Processing* module **ConvertObjectsToImage** to create an image.
-This image can be saved with the **SaveImages** module. Additionally,
-you can use the **OverlayOutlines** or **OverlayObjects** module to
-overlay outlines or objects, respectively, on a base image.
-The resulting image can also be saved with the **SaveImages** module.
-
-|
```

Comment:
This table (L24-L30) should be included before the "Measurements made by this module" section header. Restore these lines below `{HELP_ON_SAVING_OBJECTS}`.